test/Microsoft.AspNetCore.Server.KestrelTests/UvTimerHandleTests.cs

```diff
@@ -2,22 +2,23 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Server.Kestrel.Internal.Networking;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Infrastructure;
 using Microsoft.AspNetCore.Testing;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Server.KestrelTests
 {
     public class UvTimerHandleTests
     {
+        private readonly ILibuvTrace _trace = new LibuvTrace(new TestApplicationErrorLogger());
```

Comment:
Being consistent with other test classes. I don't like it 😝 But we don't really check any logs in these tests so whatever.